tests/ballerina-test/src/test/resources/test-src/connectors/sql/sql_actions_test.bal

```diff
@@ -750,6 +813,16 @@ function testINOutParameters() returns (any, any, any, any, any, any, any, any,
     :
     DIRECTION_INOUT };
 
+    if (jdbcUrl.contains("postgres")) {
+        paraTinyInt = { sqlType: sql:TYPE_SMALLINT, value: 1 };
+        paraClob = { sqlType: sql:TYPE_VARCHAR, value: "very long text", direction: sql:DIRECTION_INOUT };
+        // TODO: Until handling postgresql bytea and oid is figured out properly, just to make sure this testcase runs
+        // smoothly on postgresql, binary type is used in place of blob. Note that in "TestOutParams" also bytea type
+        // is used in place of blob type argument. Please refer
+        // https://github.com/ballerina-platform/ballerina-lang/issues/8759
+        paraBlob = { sqlType: sql:TYPE_BINARY, value: "YmxvYiBkYXRh" };
+    }
```

Comment:
Shall we remove this temporary check and properly fixing the issue will pass the test later?